index.html

```diff
@@ -45,7 +46,7 @@
                 </div>
             </nav>
 
-            <div class="tabs-wrapper" ng-show="!isActive('/about')">
+            <div class="tabs-wrapper" ng-show="isStep()">

```

Comment:
let's name this function something more descriptive, I had to go look at the definition of the function to figure out what it's doing. Should be `tabsAreVisible` or something